actionpack/lib/action_controller/metal/rendering.rb

```diff
@@ -115,7 +115,9 @@ def _normalize_text(options)
 
       # Process controller specific options, as status, content-type and location.
       def _process_options(options)
-        status, content_type, location = options.values_at(:status, :content_type, :location)
```

Comment:
Could we use `extract!` here? Or maybe we're in a hot path, if so this is fine.

Reply:
I think I'd need `extract!(status, :content_type, :location).values`, which I don't mind but would slightly prefer the three deletes. This isn't a hot path (it's from the controller so we expect this to happen about once a request), so we can do whatever feels cleanest 🙂